board/safety.h

```diff
@@ -57,6 +57,23 @@ int safety_fwd_hook(int bus_num, CAN_FIFOMailBox_TypeDef *to_fwd) {
   return current_hooks->fwd(bus_num, to_fwd);
 }
 
+// Given a CRC-8 poly, generate a static lookup table to use with a fast CRC-8
+// algorithm. Called at init time for safety modes using CRC-8.
+void gen_crc_lookup_table(uint8_t poly, uint8_t crc_lut[]) {
+  int i, j;
+
+  for (i = 0; i < 256; i++) {
```

Comment:
`for (int i = 0; i < 256; i++) {`

`i, j` can have smaller scope.

Reply:
Done